Unable to build and run from master (Maven)

With the latest master I am unable to build and start SceneBuilder. Last time I tried with Gradle, this worked out-of-the-box.

### Expected Behavior
All dependencies should be resolved and SceneBuilder should start.

### Current Behavior
The build fails with the following error:
`[ERROR] Failed to execute goal on project app: Could not resolve dependencies for project com.gluonhq.scenebuilder:app:jar:16.0.0-SNAPSHOT: Failed to collect dependencies at com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Failed to read artifact descriptor for com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Could not find artifact com.gluonhq.scenebuilder:parent:pom:16.0.0-SNAPSHOT in sonatype-snapshots (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]`

### Steps to Reproduce
Checkout latest master. Run the following command:
`mvn javafx:run -f app`

### Your Environment
Linux

### Log
```
$ mvn javafx:run -f app
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.gluonhq.scenebuilder:app:jar:16.0.0-SNAPSHOT
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] --------------------< com.gluonhq.scenebuilder:app >--------------------
[INFO] Building Scene Builder App 16.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] >>> javafx-maven-plugin:0.0.5:run (default-cli) > process-classes @ app >>>
Downloading from gluon-nexus: https://nexus.gluonhq.com/nexus/content/repositories/releases/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/maven-metadata.xml
Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/maven-metadata.xml
Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/parent-16.0.0-SNAPSHOT.pom
Downloading from gluon-nexus: https://nexus.gluonhq.com/nexus/content/repositories/releases/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/parent-16.0.0-SNAPSHOT.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.071 s
[INFO] Finished at: 2021-04-05T16:04:33+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project app: Could not resolve dependencies for project com.gluonhq.scenebuilder:app:jar:16.0.0-SNAPSHOT: Failed to collect dependencies at com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Failed to read artifact descriptor for com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Could not find artifact com.gluonhq.scenebuilder:parent:pom:16.0.0-SNAPSHOT in sonatype-snapshots (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

--- a/README.md
+++ b/README.md
@@ -48,11 +48,16 @@ It will create a partial shadow cross-platform jar under `app/target/lib/scenebu
 
 ### How to run Scene Builder ###
 
-You can run it with Maven:
+Before starting the app, all dependencies must be installed locally.
+This is achieved by:
+
+`mvn install`
+
+Then Scene Builder can be started with Maven:
 
 `mvn javafx:run -f app`
 
-or you can run the partial shadow jar, providing you have downloaded the JavaFX SDK from [here](https://gluonhq.com/products/javafx/):
+Alternatively, you can run the partial shadow jar, providing you have downloaded the JavaFX SDK from [here](https://gluonhq.com/products/javafx/):
 
 ```
 java 

